libmscore/xmlreader.cpp

```diff
@@ -520,6 +520,18 @@ Tid XmlReader::addUserTextStyle(const QString& name)
             id = Tid::USER5;
       else if (userTextStyles.size() == 5)
             id = Tid::USER6;
+      else if (userTextStyles.size() == 6)
+            id = Tid::USER7;
+      else if (userTextStyles.size() == 7)
+            id = Tid::USER8;
+      else if (userTextStyles.size() == 8)
+            id = Tid::USER9;
+      else if (userTextStyles.size() == 9)
+            id = Tid::USER10;
+      else if (userTextStyles.size() == 10)
+            id = Tid::USER11;
+      else if (userTextStyles.size() == 11)
+            id = Tid::USER12;
       else
             qDebug("too many user defined textstyles");
```

Comment:
Shouldn't there be a message box to the user here? "Some user styles could not be imported because Musescore supports a maximum of 12 user-defined styles"
Or something alike?

Reply:
Surely not here in libmscore.

Reply:
throw an error then?

Reply:
Why? We didn't before, why should we now?

Reply:
Because that's what every program does. Visio tells you when there's a problem when opening an svg and logs to a file what couldn't be imported, and it's far from the only example.

Reply:
Feel free to PR it ;-)